### DISCUSSION TITLE
can u add more then 1 plugin

### DISCUSSION THREAD

**Discussion by melvinsjostrand** [splewis/csgo-practice-mode#259] [Q&A]
Hello, my school was thinking of adding 2 of the plugins. The scrim and Pracc plugin but we cant because they both start the plugin with .setup. Can u change it somehow so we can have both the plugins?

**Comment by splewis** [comment]
Assuming "scrim plugin" means get5, this should work already. Practicemode should show up in get5's `.setup` menu as the last option (possibly on the 2nd page).

If you don't see it, make sure the plugin is loaded (run `sm plugins` in console and check if it's there; it might have encountered an error when loading and that command will report it).

**Reply by SebSchweb** [reply]
Hey is it possible to change the `.setup` to like `.setupprac` and `.setupget5` on my own. So i could use them seperate
